packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md

```diff
@@ -316,3 +334,57 @@ func retrievePushChallenges(accessToken: String) {
 ```
 
 The code assumes that each enrollment uses the same access token. Use the appropriate access token if your app handles multiple user accounts.
+
+Alternatively, you can retrieve undelivered challenges using the [MyAccount App Authenticators API](https://developer.okta.com/docs/api/openapi/okta-myaccount/myaccount/tag/AppAuthenticator/#tag/AppAuthenticator/operation/listAppAuthenticatorPendingPushNotificationChallenges).
+
+## Access token management
+
+> **Note:** To enable the JWT bearer grant type:
+>  * Send a PUT request to `/apps/{appId}`. Ensure that the `grant_types` array contains the following string:
+>    `urn:ietf:params:oauth:grant-type:jwt-bearer`
+>  * If you use custom authorization servers, update the policy rules to update the grant type.
+
+The SDK communicates with an Okta server using HTTPS protocol and requires an access token for user authentication and authorization. For authentication flows and requesting access tokens, use the latest version of the [Okta Swift mobile SDK](https://github.com/okta/okta-mobile-swift). To enroll a push authenticator, the access token needs the `okta.myAccount.appAuthenticator.manage` scope. You can also use this scope for the following operations:
+
+* Enroll and unenroll user verification keys
+* Update device token for push authenticator enrollment
+* Request pending push challenges
+* Enable and disable CIBA capability for push authenticator enrollment
+* Delete push authenticator enrollment
+  < **Note:** Applications that use sensitive data shouldn't store or cache access tokens or refresh access tokens that contain the `okta.myAccount.appAuthenticator.manage` scope. Instead, reauthenticate the user and get a new access token.
+
+  High risk operations include the following:
```

Comment:
Two spaces in front of this line makes it look like a subset of the previous list.

Do we need to tell the user that high risk operations need reauthentication?
For example: "The following is a list of operations that are considered high risk and require reauthentication:"